tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932236.yaml

```diff
@@ -1463,3 +1463,147 @@ tests:
+          version: HTTP/1.1
+        output:
+          log:
+            no_expect_ids: [932236]
+  - test_id: 83
+    desc: "False positive with UUID"
+    stages:
+      - input:
+          dest_addr: 127.0.0.1
+          headers:
+            Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
+            Host: localhost
+            User-Agent: "OWASP CRS test agent"
+          method: POST
+          port: 80
+          uri: "/post"
+          version: HTTP/1.1
+          data: |
+            uuid=dfc987c2-72e2-4a8e-ad98-e0bf1bc3a01c
+        output:
+          log:
+            no_expect_ids: [932236]
+  - test_id: 84
+    desc: "False positive with word idendity"
+    stages:
+      - input:
+          dest_addr: 127.0.0.1
+          headers:
+            Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
+            Host: localhost
+            User-Agent: "OWASP CRS test agent"
+          method: POST
+          port: 80
+          uri: "/post"
+          version: HTTP/1.1
+          data: |
+            uuid=idendity
+        output:
+          log:
+            no_expect_ids: [932236]
+  - test_id: 85
+    desc: "False positive with word unique"
+    stages:
+      - input:
+          dest_addr: 127.0.0.1
+          headers:
+            Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
+            Host: localhost
+            User-Agent: "OWASP CRS test agent"
+          method: POST
+          port: 80
+          uri: "/post"
+          version: HTTP/1.1
+          data: |
+            uuid=unique
+        output:
+          log:
+            no_expect_ids: [932236]
+  - test_id: 86
+    desc: "Block executing id comand no arguments"
+    stages:
+      - input:
+          dest_addr: 127.0.0.1
+          headers:
+            Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
+            Host: localhost
+            User-Agent: "OWASP CRS test agent"
+          method: POST
+          port: 80
+          uri: "/post"
+          version: HTTP/1.1
+          data: |
+            code=id
+        output:
+          log:
+            expect_ids: [932236]
+  - test_id: 87
+    desc: "false positive with Nextcloud Mail widget"
+    stages:
+      - input:
+          dest_addr: 127.0.0.1
+          headers:
+            Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
+            Host: localhost
+            User-Agent: "OWASP CRS test agent"
+          method: POST
+          port: 80
+          uri: "/post"
+          version: HTTP/1.1
+          data: |
+            code=mail-unread
+        output:
+          log:
+            no_expect_ids: [932236]
+  - test_id: 88
+    desc: |
+      false positive with WordPress order posts by date
```

Comment:
```suggestion
      False positive with WordPress order posts by date
```

Reply:
The word that matched as false positive was `post`, right? The description should mention that.

Reply:
It's just ``date`` that's being matched. Ordering posts in WordPress by date